source/dynamical_systems/src/DynamicalSystem.cpp

```diff
@@ -14,39 +15,57 @@ template<>
 DynamicalSystem<JointState>::DynamicalSystem() : base_frame_(JointState()) {}
 
 template<>
-DynamicalSystem<CartesianState>::DynamicalSystem() : base_frame_(CartesianState::Identity("world")) {}
+DynamicalSystem<CartesianState>::DynamicalSystem() : base_frame_(CartesianState()) {}
 
 template<>
 DynamicalSystem<CartesianState>::DynamicalSystem(const std::string& base_frame) :
     base_frame_(CartesianState::Identity(base_frame, base_frame)) {}
 
 template<class S>
-bool DynamicalSystem<S>::is_compatible(const S&) {
+bool DynamicalSystem<S>::is_compatible(const S&) const {
```

Comment:
Ah i just saw that this function is `virtual`, should I `override` it?

Reply:
well no, because it's declared virtual in this class, there is no ancestor to override!